cpp/ql/lib/semmle/code/cpp/models/implementations/Iterator.qll

```diff
@@ -223,7 +223,7 @@ private class IteratorCrementMemberOperator extends MemberFunction, DataFlowFunc
     output.isQualifierObject()
     or
     input.isQualifierObject() and
-    output.isReturnValueDeref()
+    output.isReturnValue()
```

Comment:
Hm, yes. This might be an issue with how the use-use flow PR did flow through iterators (because they're not actually a pointer type, but we want to model them as such). I think I'll revert this change (+ the one in `StdStringAppend` since it's also related to iterators). It may mean that we have less flow in the use-use code, but since this is now in experimental it doesn't really matter. And I'd rather risk losing flow in experimental than in our production code.